galley/pkg/config/analysis/analyzers/analyzers_test.go

```diff
@@ -323,6 +323,14 @@ func TestAnalyzersHaveUniqueNames(t *testing.T) {
 	}
 }
 
+func TestAnalyzersHaveDescription(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	for _, a := range All() {
+		g.Expect(a.Metadata().Description).ToNot(Equal(""))
+	}
+}
```

Comment:
Please also update the analyzers README to include this new requirement.

Reply:
Done.